doc/pages/keys.asciidoc

```diff
@@ -520,36 +514,36 @@ to skim through the jump list using:
     jump backward
 
 *<c-s>*::
-    save current selections
+    save selections
 
 == Multiple selections
 
 *s*::
-    create a selection
+    create a selection with the given regex
```

Comment:
`<s>` can create more than one selection, so how about: "create a selection for each match of the given regex"